tests/test_msg_cosmos_getaddress.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import common
+
+import keepkeylib.messages_pb2 as proto
+import keepkeylib.types_pb2 as proto_types
+import keepkeylib.messages_cosmos_pb2 as cosmos_proto
+from keepkeylib.client import CallException
+from keepkeylib.tools import parse_path
+
+DEFAULT_BIP32_PATH = "m/44h/118h/0h/0/0"
+
+class TestMsgCosmosGetAddress(common.KeepKeyTest):
+    def test_cosmos_get_address(self):
+        self.setup_mnemonic_nopin_nopassphrase()
+        address = self.client.cosmos_get_address(parse_path(DEFAULT_BIP32_PATH))
+        assert address.address == "cosmos15cenya0tr7nm3tz2wn3h3zwkht2rxrq7q7h3dj"
+
+    def test_cosmos_get_address_sep(self):
+        self.mnemonic12 = 'illness spike retreat truth genius clock brain pass fit cave bargain toe'
```

Comment:
IIUC, this seed phrase came from some Stellar SEP. It would be better to use test vectors from the Cosmos ecosystem. Maybe @BitHighlander knows of some?